src/containers/createEvent/ducks/types.ts

```diff
@@ -0,0 +1,12 @@
+export interface NewEventInformation {
```

Comment:
This seems to exactly duplicated the other interface, minus ID?

Reply:
Yes, but making ID an optional field in the original interface broke a lot of things in other files and it seemed like a lot more work to change all of that. I can do that if needed tho